beta/src/components/MDX/MDXComponents.tsx

```diff
@@ -353,15 +353,18 @@ function InlineTocItem({items}: {items: Array<NestedTocNode>}) {
   );
 }
 
-function LinkWithTodo({href, children, ...props}: JSX.IntrinsicElements['a']) {
```

Comment:
+1 but I think there are a couple TODO links left, want to add the links before deleting this (otherwise the links will be broken)?

Reply:
[Here](https://github.com/reactjs/reactjs.org/blame/ef41566f7664d6e8a68ff46dfbd7fba4750887f4/beta/src/content/learn/responding-to-events.md#L234) and [Here](https://github.com/reactjs/reactjs.org/blame/c79307666bb02b8d74d60c413cf41ec6d54c97ca/beta/src/content/learn/writing-markup-with-jsx.md#L217)

Reply:
oh i thought i fixed those

Reply:
okay moment